lib/cell/concept.rb

```diff
@@ -5,8 +5,23 @@ class Concept < Cell::ViewModel
 
     # TODO: this should be in Helper or something. this should be the only entry point from controller/view.
     class << self
-      def class_from_cell_name(name)
-        name.camelize.constantize
+      def class_from_cell_name(name, controller)
+        @cell_class ||= begin
+          supports_path = controller.supports_path?
+          routes  = controller.respond_to?(:_routes)  && controller._routes
+          helpers = controller.respond_to?(:_helpers) && controller._helpers
+
+          Class.new(name.camelize.constantize) do
+            if routes
+              include routes.url_helpers(supports_path)
+              include routes.mounted_helpers
+            end
+
+            if helpers
+              include helpers
```

Comment:
This includes should not happen for all cells. You should build a `Cell::Rails::Helpers::Url` module